.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "lumper"]
```

Comment:
I'm fine with submodules, but I want to warn you that working on a submodule and maintaining/submitting changes to lambdo may not be trivial. I expect that you have compared that option with having an actual fork as a local crate. Again, either way is fine with me.

Reply:
We had a few words about that and we finally agree that submodules are a good option.